templates/nodejs/config/common.yml

```diff
@@ -40,12 +40,13 @@ default:
       brokers:
         - {{ asyncapi.server(params.server).url() | host }}
       consumerOptions:
-        groupId: 'replace-me'
+        groupId: {{ asyncapi.info().title() | camelCase }}
```

Comment:
Sorry for including an unrelated change in this PR. I'm just trying to reduce overhead.